docs/CONTRIBUTING-ta.md

```diff
@@ -0,0 +1,265 @@
+
+#### வகைகள்
+
+ஒரு வளம் எந்த பட்டியலுக்குச் சொந்தமானது என்பதைத் தீர்மானிப்பதற்கான முதல் விதி, வளம் தன்னை எவ்வாறு விவரிக்கிறது என்பதைப் பார்ப்பதாகும். அது தன்னை ஒரு புத்தகம் என்று அழைத்தால், அது ஒரு புத்தகமாக இருக்கலாம்.
+
+#### நாங்கள் பட்டியலிடாத வகைகள்
+
+இணையம் பரந்த அளவில் இருப்பதால், நாங்கள் எங்கள் பட்டியல்களில் சேர்க்கவில்லை:
+
+- வலைப்பதிவுகள்
+- வலைப்பதிவு பதிவுகள்
+- கட்டுரைகள்
+- வலைத்தளங்கள் (except for those that host LOTS of items that we list).
+- பாடங்கள் அல்லது திரைக்காட்சிகள் அல்லாத வீடியோக்கள்.
+- புத்தக அத்தியாயங்கள்
+- புத்தகங்களிலிருந்து டீஸர் மாதிரிகள்
+- ஐஆர்சி அல்லது டெலிகிராம் சேனல்கள்
+- ஸ்லாக்ஸ் அல்லது அஞ்சல் பட்டியல்கள்
+
+எங்கள் போட்டி நிரலாக்க பட்டியல்கள் இந்த விலக்குகளைப் பற்றி அவ்வளவு கண்டிப்பானவை அல்ல. ரெப்போவின் நோக்கம் சமூகத்தால் தீர்மானிக்கப்படுகிறது; நீங்கள் ஒரு மாற்றத்தை அல்லது நோக்கத்துடன் கூடுதலாக பரிந்துரைக்க விரும்பினால், தயவுசெய்து ஆலோசனையைச் செய்ய ஒரு சிக்கலைப் பயன்படுத்தவும்.
+
+
+### புத்தகங்கள் எதிராக பிற பொருட்கள்
+
+புத்தகத் திறனைப் பற்றி நாங்கள் அவ்வளவு பரபரப்பாக இல்லை. ஒரு வளம் ஒரு புத்தகம் என்பதைக் குறிக்கும் சில பண்புக்கூறுகள் இங்கே:
+
+- இதில் ISBN (International Standard Book Number) உள்ளது
+- இதில் உள்ளடக்க அட்டவணை உள்ளது
+- பதிவிறக்கம் செய்யக்கூடிய பதிப்பு வழங்கப்படுகிறது, குறிப்பாக ePub கோப்புகள்
+- இது பதிப்புகளைக் கொண்டுள்ளது
+- இது ஊடாடும் உள்ளடக்கம் அல்லது வீடியோக்களைப் பொறுத்தது அல்ல
+- இது ஒரு தலைப்பை விரிவாக மறைக்க முயற்சிக்கிறது
+- இது தன்னிறைவு பெற்றது
+
+இந்த பண்புகளைக் கொண்டிராத நிறைய புத்தகங்களை நாம் பட்டியலிடுகிறோம்; அது சூழலைப் பொறுத்தது.
+
+#### புத்தகங்கள் எதிராக. படிப்புகள்
+
+சில நேரங்களில் இவற்றை வேறுபடுத்துவது கடினமாக இருக்கலாம்!
+
+படிப்புகள் பெரும்பாலும் தொடர்புடைய பாடப்புத்தகங்களைக் கொண்டுள்ளன, அவற்றை நாங்கள் எங்கள் புத்தகப் பட்டியல்களில் பட்டியலிடுவோம். படிப்புகளில் விரிவுரைகள், பயிற்சிகள், சோதனைகள், குறிப்புகள் அல்லது பிற போதனை உதவிகள் உள்ளன. ஒரு சொற்பொழிவு அல்லது வீடியோ என்பது ஒரு பாடநெறி அல்ல. ஒரு பவர்பாயிண்ட் ஒரு பாடநெறி அல்ல.
+
+#### ஊடாடும் பயிற்சிகள் எதிராக பிற விஷயங்கள்
+
+நீங்கள் அதை அச்சிட்டு அதன் சாரத்தை தக்க வைத்துக் கொள்ள முடிந்தால், அது ஒரு ஊடாடும் பயிற்சி அல்ல.
+
+
+## ஆட்டோமேஷன்
+
+- வடிவமைப்பு விதிகள் அமலாக்கம் [GitHub Actions](https://github.com/features/action) வழியாக தானியங்கி செய்யப்படுகிறது [fpb-lint](https://github.com/vhf/free-programming-books-lint) ஐப் பயன்படுத்தி (பார்க்க [`.github/workflows/fpb-lint.yml`](../.github/workflows/fpb-lint.yml)).
+  
+- URL சரிபார்ப்பு [amazing_bot](https://github.com/dkamsing/awesome_bot) ஐ பயன்படுத்துகிறது.
+  
+- URL சரிபார்ப்பைத் தூண்டுவதற்கு, 'check_urls = file_to_check' ஐக் கொண்ட ஒரு கமிட் செய்தியை உள்ளடக்கிய ஒரு கமிட்டை அழுத்தவும்:
+
+    ```bash
+    சிறப்பு check_urls = free-programming-books.md free-programming-books-en.md
+    ```
```

Comment:
A bash script may not use 'Tamil' language. Please review your changes once again. :)